wallet/src/de/schildbach/wallet/data/BlockchainStateRoomConverters.kt

```diff
@@ -20,17 +20,7 @@ package de.schildbach.wallet.data
 import androidx.room.TypeConverter
 import java.util.*
 
-class RoomConverters {
-
-    @TypeConverter
-    fun fromTimestamp(value: Long?): Date? {
-        return value?.let { Date(it) }
-    }
-
-    @TypeConverter
-    fun dateToTimestamp(date: Date?): Long? {
-        return date?.time
-    }
+class BlockchainStateRoomConverters {
 
     @TypeConverter
     fun fromImpedimentsEnumSet(value: String?): Set<BlockchainState.Impediment> {
```

Comment:
Converters are split into a general file and a `BlockchainState`-related file for single-responsibility and convenience.